include/stack-graphs.h

```diff
@@ -0,0 +1,121 @@
+#ifndef STACK_GRAPHS_H_
```

Comment:
I'm purposefully _not_ marking this as a generated according to linguist/git, so that we can see what changes to the C API look like during PR review.